README.md

```diff
@@ -22,7 +22,18 @@ are normally:
 
 - `KUBERNETES_BRANCH`: The kubernetes branch to get OpenAPI spec from. e.g. "master"
 - `CLIENT_VERSION`: Client version string. e.g. "1.0.0b1"
-- `PACKAGE_NAME`: Package name for the generated client. e.g. "kubernetes"
+- `PACKAGE_NAME`: Package name for the generated client. e.g. "kubernetes" 
+
+Example settings file for python-client:
+
+export KUBERNETES_BRANCH="master" 
```

Comment:
Is there a stray character in this line?

Reply:
Thanks, there was an extra space character at the end of that line. It's removed now.

Reply:
I still see some weird "L Sep" unicode character in the diff here?

Sorry to be so picky, but I want to keep the change clean.

Reply:
I can't see it. Which line is it on? Thanks.